### PR TITLE
Updated Kubernetes-Client Version to 4.7.1

### PR DESCRIPTION
**Pull request by dev-gaur**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -43,6 +42,8 @@ Usage:
 * Doc: Add documentation #38
 * Add Sample Quickstarts to project #33
 * Fix: Run tests for current version #35
+* Updated Kubernetes-Client Version to 4.7.1
+* Refactor #29: Add Maven Enforcer Plugin
```
Comment (rohanKanojia) [inline CHANGELOG.md]: Why is this here?

File: jkube-kit/enricher/api/src/main/java/org/eclipse/jkube/maven/enricher/api/visitor/MetadataVisitor.java [hunk 2]
```diff
@@ -210,7 +210,9 @@ protected Kind getKind() {
 
         @Override
         protected ObjectMeta getOrCreateMetadata(ReplicationControllerBuilder item) {
-            return item.hasMetadata() ? item.buildMetadata() : item.withNewMetadata().endMetadata().buildMetadata();
+            //return item.hasMetadata() ? item.buildMetadata() : item.withNewMetadata().endMetadata().buildMetadata();
```
Comment (rohanKanojia) [inline jkube-kit/enricher/api/src/main/java/org/eclipse/jkube/maven/enricher/api/visitor/MetadataVisitor.java]: remove these commented section

File: jkube-kit/parent/pom.xml [hunk 3]
```diff
@@ -57,7 +57,7 @@
     <version.json-path-assert>2.2.0</version.json-path-assert>
     <version.jsr305>3.0.2</version.jsr305>
     <version.JUnitParams>1.0.5</version.JUnitParams>
-    <version.kubernetes-client>4.6.2</version.kubernetes-client>
+    <version.kubernetes-client>4.7.1</version.kubernetes-client>
```
Comment (rohanKanojia) [inline jkube-kit/parent/pom.xml]: Shall we upgrade to 4.8.0 instead?

Comment (manusa) [reply jkube-kit/parent/pom.xml]: As JKube requires CQ to upgrade a dependency, I'd merge this one once the rest of the comments are fine.
Then do the upgrade to 4.8.0 in the scope of another issue.
Else we'll be stuck forever.

**Comment by dev-gaur** [comment]
http://dev.eclipse.org/ipzilla/show_bug.cgi?id=21606
http://dev.eclipse.org/ipzilla/show_bug.cgi?id=21605

CQs have been resolved. Shall we proceed to merge?

File: jkube-kit/enricher/api/src/main/java/org/eclipse/jkube/maven/enricher/handler/NamespaceHandler.java [hunk 4]
```diff
@@ -15,6 +15,7 @@
 
 import io.fabric8.kubernetes.api.model.Namespace;
 import io.fabric8.kubernetes.api.model.NamespaceStatus;
```
Comment (rohanKanojia) [inline jkube-kit/enricher/api/src/main/java/org/eclipse/jkube/maven/enricher/handler/NamespaceHandler.java]: Could you please remove this?

Comment (dev-gaur) [reply jkube-kit/enricher/api/src/main/java/org/eclipse/jkube/maven/enricher/handler/NamespaceHandler.java]: done

**Comment by manusa** [comment]
Relates to: #50
